src/main/java/org/myrobotlab/service/config/Gpt3Config.java

```diff
@@ -13,9 +13,9 @@ public class Gpt3Config extends ServiceConfig {
 
   public int maxTokens = 256;
   public float temperature = 0.7f;
-  public String url = "https://api.openai.com/v1/completions";
+  public String url = "https://api.openai.com/v1/chat/completions";
```

Comment:
Does changing this also change the default YAML config file?

Reply:
yes it does